blockchain_connector/Dockerfile

```diff
@@ -124,25 +130,18 @@ RUN mkdir -p build \
 WORKDIR /project/avalon/common/python
 
 RUN echo "Building Avalon Common Python\n" \
-  && python setup.py bdist_wheel \
-  && echo "Installing Avalon Common Python" \
-  && pip3 install dist/*.whl
+  && make
 
 #install avalon crypto module
 WORKDIR /project/avalon/common/crypto_utils
 
 RUN echo "Building Avalon Common Crypto Python\n" \
-  && make \
-  && echo "Installing Avalon Common Crypto Python" \
-  && pip3 install dist/*.whl
-
+  && make
```

Comment:
Are we using crypto functionalities in block chain connector ?

Reply:
Not looking into it as of now for this lint specific PR.